Gruntfile.js

```diff
@@ -50,6 +80,44 @@ module.exports = function (grunt) {
         });
     };
 
+    // Base distribution configuration
+    // This should specify only the options for include/exclude;
+    // a parallel "minify" set with options.compress: true
+    // will be generated for the final configuration
+    var baseDistributions =
```

Comment:
OK - this is a reasonable preference. I'll dig more into Grunt to figure out how to accomplish this in a more Grunt-like way.